README.md

```diff
@@ -35,130 +31,70 @@ Welcome to the Netlify CLI! The new 2.0 version (now in beta) was rebuilt from t
-```
-
-### CI and Environment Variables.
-
-The following environment variables can be used to override configuration file lookups and prompts:
-
-- `NETLIFY_AUTH_TOKEN` - an access token to use when authenticating commands. **KEEP THIS VALUE PRIVATE**
-- `NETLIFY_SITE_ID` - override any linked site in the current working directory.
-
-## Getting Started + Docs
-
-Please see the getting started guide on our docs website:
-
-- [netlify.com/docs/cli](https://www.netlify.com/docs/cli)
-
-<!-- TODO: Grab a screenshot to make it pop more -->
-
-## Full Command Reference
+## Commands
 
 <!-- AUTO-GENERATED-CONTENT:START (GENERATE_COMMANDS_LIST) -->
-### [deploy](/docs/commands/deploy.md)
+### [deploy](/docs/deploy.md)
 
 Create a new deploy from the contents of a folder
 
-### [init](/docs/commands/init.md)
+### [init](/docs/init.md)
 
 Configure continuous deployment for a new or existing site
 
-### [link](/docs/commands/link.md)
+### [link](/docs/link.md)
 
 Link a local repo or project folder to an existing site on Netlify
 
-### [login](/docs/commands/login.md)
+### [login](/docs/login.md)
 
 Login to your Netlify account
 
-### [logout](/docs/commands/logout.md)
+### [logout](/docs/logout.md)
 
 Logout of your Netlify account
 
-### [open](/docs/commands/open.md)
+### [open](/docs/open.md)
 
 Open settings for the site linked to the current folder
 
 | Subcommand | description  |
 |:--------------------------- |:-----|
-| [`open:admin`](/docs/commands/open.md#openadmin) | Opens current site admin UI in Netlify  |
-| [`open:site`](/docs/commands/open.md#opensite) | Opens current site url in browser  |
+| [`open:admin`](/docs/open.md#openadmin) | Opens current site admin UI in Netlify  |
+| [`open:site`](/docs/open.md#opensite) | Opens current site url in browser  |
 
 
-### [sites](/docs/commands/sites.md)
+### [sites](/docs/sites.md)
 
 Handle various site operations
 
 | Subcommand | description  |
 |:--------------------------- |:-----|
-| [`sites:create`](/docs/commands/sites.md#sitescreate) | Create an empty site (advanced)  |
-| [`sites:list`](/docs/commands/sites.md#siteslist) | List all sites you have access too  |
+| [`sites:create`](/docs/sites.md#sitescreate) | Create an empty site (advanced)  |
+| [`sites:list`](/docs/sites.md#siteslist) | List all sites you have access too  |
 
 
-### [status](/docs/commands/status.md)
+### [status](/docs/status.md)
 
 Print status information
 
 | Subcommand | description  |
 |:--------------------------- |:-----|
-| [`status:hooks`](/docs/commands/status.md#statushooks) | Print hook information of the linked site  |
+| [`status:hooks`](/docs/status.md#statushooks) | Print hook information of the linked site  |
 
 
-### [unlink](/docs/commands/unlink.md)
+### [unlink](/docs/unlink.md)
 
 Unlink a local folder from a Netlify site
 
-### [watch](/docs/commands/watch.md)
+### [watch](/docs/watch.md)
 
 Watch for site deploy to finish
 
 
 <!-- AUTO-GENERATED-CONTENT:END -->
 
-### telemetry
```

Comment:
Whats the reason for removing this?

Reply:
I moved it into the main doc: https://deploy-preview-1786--www.netlify.com/docs/cli/#usage-data-collection

Reply:
Ahh yeah, seeing that now.  Fine with me.